docs/examples/index.md

```diff
@@ -3,6 +3,109 @@ title: Basic usage
 sidebar_label: Basic usage
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+# Code examples
 
-There is nothing here yet.
+## Different ways how to pass an input
+
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+// Run the actor and wait for it to finish up to 60 seconds. Input is not persisted for next runs.
+const runData = await actorClient.call({ hashtags: ['twitter'], resultsLimit: 20 }, { waitSecs: 60 });
+
```

Comment:
looks like you keep the blank like here on purpose. i dont think it should be there.